sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlanner.scala

```diff
@@ -31,6 +31,12 @@ class SparkPlanner(
     val experimentalMethods: ExperimentalMethods)
   extends SparkStrategies {
 
+  private val EXPENSIVE_EXPR_PREFIX = "expensive_col_"
+
+  def extractExpensiveExprs(e: Expression): Seq[Expression] = e.collect {
+    case gjo: GetJsonObject => gjo
```

Comment:
If you handle only `GetJsonObject`, please narrow down the PR title exactly.

Reply:
> If you handle only `GetJsonObject`, please narrow down the PR title exactly.

Sorry for forgot to add tag `[WIP]`. Since I am not familiar with all expensive expr or function. 
In our env,  many data stored as Json, we use this a lot and always expr with `get_json_object` is slow.

So I made this and hope for some advise and to see if it is reasonable enough